types/ember/index.d.ts

```diff
@@ -2113,7 +2113,15 @@ declare module 'ember' {
                 options?: { path?: string; resetNamespace?: boolean },
                 callback?: (this: RouterDSL) => void
             ): void;
-            mount(name: string): void;
+            mount(
+                name: string,
+                options?: {
```

Comment:
One quick note for reference: with types like this, you can use the `Partial<T>` type from the TS library:

```ts
mount(
    name: string,
    options?: Partial<{
        as: string,
        path: string,
        resetNamespace: boolean,
        engineInfo: any
    }>
): void;
```

That lets you get the benefit of having all the items in the object be optional, without having to add `?` to every one of them. It's particularly convenient for times where the options might change in the future, but would all always be optional.

(Here, it's good either way; I just figured I'd add the comment since many people don't know about `Partial<T>`!)

Reply:
Thank you so much! I did not know that! You are awesome!